iroh-net/src/lib.rs

```diff
@@ -253,6 +251,10 @@ pub(crate) mod util;
 
 pub use endpoint::{AddrInfo, Endpoint, NodeAddr};
 pub use iroh_base::{key, key::NodeId};
+// TODO(@divma): re-evaluate this re-export
+pub use netwatch as net;
+// TODO(@divma): re-evaluate this re-export
```

Comment:
I would vote to not add the reexports, the goal is to reduce the API surface of `iroh-net`

Reply:
here it is :) no problem